lib/cdo/aws/s3.rb

```diff
@@ -209,6 +209,31 @@ def self.process_file(bucket, key)
       end
     end
 
+    # Finds all objects with a given extension in the given bucket
+    # @param bucket [String] The S3 bucket name.
+    # @param ext [String] An extension to search for.
+    # @param prefix [String] An optional prefix to filter objects by.
+    def self.find_objects_with_ext(bucket, ext, prefix)
+      object_keys = []
+      # list_objects is paginated and each response is up to 1000 objects
+      create_client.list_objects_v2(bucket: bucket, prefix: prefix).each do |response|
```

Comment:
These 2 methods are great additions to our s3 wrapper library. Should we include tests for them? I think they would go here? https://github.com/code-dot-org/code-dot-org/blob/staging/shared/test/test_aws_s3_integration.rb

Reply:
Good idea! I didn't see this file when I did a quick search for existing tests, I'll add some

Reply:
Might be overkill. Stubbing AWS API calls is tedious.

Reply:
i think the rename is not worth testing, since its just api calls, but i'll add one for the find since it has some complex logic

Reply:
good thing i wrote that test, there was a type 😬